public/data/devesh2511.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Devesh Jain",
+  "type": "personal",
+  "bio": "I build things that can't be touched, but can be felt, can be used, can be helpful.. Committing codes and mistakes..",
+  "avatar": "https://github.com/devesh2511.png",
+  "links": [
+    
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/devesh-jain-2511dj/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Check out my GitHub",
```

Comment:
> Kindly, if possible change this line.

```suggestion
      "name": "Follow me on GitHub",
```

Reply:
@syedsohan They can word it how they want, there's no need for them to change that line.

Reply:
> @syedsohan They can word it how they want, there's no need for them to change that line.

Oh ok! I asked to change just because of it was mentioned in the README.md! @emmalearnscode 🙂

Reply:
@syedsohan The readme is just an example, mainly to show how to structure the code but you're free to write whatever you like :)

Reply:
Ok 🙂 @emmalearnscode